cleanup.yaml

```diff
@@ -14,119 +15,63 @@
 
 
 - name: Delete koji
-  hosts: osbs_box_host
+  hosts: localhost
   gather_facts: false
   tags:
     - koji
-    # All tags related to koji PVs (deleting PVs while in use is a bad idea)
-    - pvs
-    - koji_pvs
-    - koji_db_pv
-    - koji_files_pv
     # One tag to rule them all
     - everything
 
   tasks:
-    - name: Delete koji namespace from openshift
-      command: oc delete namespace "{{ koji_namespace }}"
-      register: delete_koji_namespace
-      changed_when: delete_koji_namespace.rc == 0
-      failed_when: delete_koji_namespace.rc != 0 and
-                   'not found' not in delete_koji_namespace.stderr
-      when: not oc_login.failed
-
-    - name: Delete koji PVs from openshift
-      command: oc delete pv --selector volume=koji-volume
-      register: delete_koji_pvs
-      changed_when: delete_koji_pvs.rc == 0 and
-                    delete_koji_pvs.stdout != 'No resources found'
-      failed_when: delete_koji_pvs.rc != 0
+    - name: Delete koji project from openshift
+      command: oc delete project/"{{ koji_project }}"
+      register: delete_koji_project
+      changed_when: delete_koji_project.rc == 0
+      failed_when: delete_koji_project.rc != 0 and
+                   'cannot delete resource "projects"' not in delete_koji_project.stderr
```

Comment:
Should this not be
`'cannot delete resource "projects"' in delete_koji_project.stderr` 
as when this message IS in the stderr, it defines failure? similiar in other instances

Reply:
That's to cover the case where (as sometimes happens) the playbook is able to delete a Project, but is interrupted (or whatever) and fails to delete others. When the play is re-run, it will fail and exit because of that error on the already-deleted Project, thus skipping everything after that deletion, so we ignore it.

Reply:
ack